.changes/v2.14.0/403-features.md

```diff
@@ -0,0 +1,2 @@
+* Added type `NsxtAlbConfig` and functions `NsxtEdgeGateway.UpdateAlbSettings`, `NsxtEdgeGateway.GetAlbSettings`,
+  `NsxtEdgeGateway.DisableAlb` [GH-403]
```

Comment:
Maybe worth mentioning the helper function too?

Reply:
The helper function is a private one made for testing that should not be mentioned in the changes log.

Reply:
Yep, I am on this one with Giuseppe - it is an internal function used for testing only.